article_netmgr.hh

```diff
@@ -152,11 +122,12 @@ public:
   sptr< Dictionary::DataRequest > getResource( QUrl const & url,
                                                QString & contentType );
 
-protected:
+// protected:
```

Comment:
Commented out code again. Please make certain that all member functions below this label need to be `public`. If yes, remove this line. Otherwise, restore it or even change to `private`.